adam-pipeline/workflow.wdl

```diff
@@ -0,0 +1,22 @@
+task adam {
+  File in
+  File knownSites
+  String mem
+
+  command {
+    python /opt/adam-pipeline/wrapper.py --known-sites ${knownSites} --sample ${in} --output `pwd`/small.processed.bam --memory ${mem}
```

Comment:
Agreed.  But I do like that the WDL then hides this detail from me.  I can just focus on inputs/outputs.  The key would be to make sure to clearly associate a release of the Docker image with the correct release of the WDL... if only there was a site that associated Docker images with WDL/CWL descriptors :-)